code/injections/storage/dummy/stored_countries.go

```diff
@@ -182,7 +182,200 @@ var (
+				},
+				{
+					ImageURL:    util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/7/7d/New_Zealand_road_sign_R1-2_%2860%29.svg/1024px-New_Zealand_road_sign_R1-2_%2860%29.svg.png"),
+					Description: util.StrPtr("Variable speed sign. Can also have black background"),
+				},
+				{
+					ImageURL:     util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/9/99/Products-ez-szsk144-rs62-w16-5-1-kura-school-speed-30km-sign-thumb.jpg"),
+					Description:  util.StrPtr("Maori Te Kura school zone sign"),
+					Significance: util.StrPtr("There is only one Te Kura school in the South of NZ. The rest are in the North."),
+					// RegionSpecificity:  float{0.4},
+					// CountrySpecificity: float32{1},
+				},
+			},
+			// Chevrons: []core.Hint{
+			// 	// {
+			// 	// 	ImageURL: util.StrPtr("https://calico-cut-geo-tools.s3.amazonaws.com/argentina-chevron-blue.png"),
+			// 	// },
+			// 	// {
+			// 	// 	ImageURL: util.StrPtr("https://calico-cut-geo-tools.s3.amazonaws.com/argentina-chevron-left-red.png"),
+			// 	// },
+			// 	// {
+			// 	// 	ImageURL: util.StrPtr("https://calico-cut-geo-tools.s3.amazonaws.com/argentina-chevron-right-red.png"),
+			// 	// },
+			// 	// {
+			// 	// 	ImageURL: util.StrPtr("https://calico-cut-geo-tools.s3.amazonaws.com/argentina-chevron-yellow-black.png"),
+			// 	// },
+			// },
+			// Highway: []core.Hint{
+			// 	// {
+			// 	// 	ImageURL: util.StrPtr("https://calico-cut-geo-tools.s3.amazonaws.com/argentina-highway.png"),
+			// 	// },
+			// },
+			// Street: []core.Hint{
+			// 	// {
+			// 	// 	ImageURL:    util.StrPtr("https://calico-cut-geo-tools.s3.amazonaws.com/argentina-street-name-sign.png"),
+			// 	// 	Description: util.StrPtr("Street name signs are dark and perpendicular"),
+			// 	// },
+			// },
+			// Transit: []core.Hint{
+			// 	// {
+			// 	// 	ImageURL:    util.StrPtr("https://calico-cut-geo-tools.s3.amazonaws.com/argentina-bus-sign.png"),
+			// 	// 	Description: util.StrPtr("Bus stop sign"),
+			// 	// },
+			// },
+		},
+		AreaCodes: []core.AreaCode{
+			{
+				Code:     "9",
+				Location: "Auckland/North Island - North/Pokeno/Whangarei",
+			},
+			{
+				Code:     "3",
+				Location: "Christchurch/Dunedin/Invercargill/Nelson/South Island/Timaru",
+			},
+			{
+				Code:     "7",
+				Location: "Hamilton/North Island - Central and East/Rotorua/Tauranga",
+			},
+			{
+				Code:     "6",
+				Location: "Hastings/Napier/New Plymouth/North Island - West/Palmerston North/Wanganui",
+			},
+			{
+				Code:     "24",
+				Location: "New Zealand External Territories",
+			},
+			{
+				Code:     "4",
+				Location: "Wellington Region",
+			},
+		},
+		UtilityPoles: &core.UtilityPoles{
+			// ExtraNotes: []string{},
+			Poles: []core.Hint{
+				{
+					ImageURL:    util.StrPtr("https://geohints.com/Sources/Poles/pole_37.jpg"),
+					Description: util.StrPtr("Has a depression and white bottom"),
+					// Tags: ["concrete", "grey"],
```

Comment:
`Tags: []string{"concrete", "grey"},`

also as a heads up, this structure is called a "slice" in Go. might help with searching them in the future. Go technically has arrays, but nobody really uses them directly, they're more so used under the hood for slices.